integrated_channels/canvas/models.py

```diff
@@ -67,6 +67,18 @@ class CanvasEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguratio
     .. no_pii:
     """
 
+    client_id = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+        editable=False,
```

Comment:
why is editable False here? According to docs this means it won't show up in Django Admin, but don't we need that to be the case for ECS to eventually enter this in for the customer? Also not sure we should allow blanks. Null is fine though.

Reply:
yeah think I should not have `editable=False`, no reason to since we do want to edit it in forms eventually ... will remove

since we have no data right now (including blanks) in the db, it should be fine to remove `blank=True` as well, agreed on, let's not allow blanks here